packages/react-ui/src/components/atoms/PasswordField.js

```diff
@@ -0,0 +1,35 @@
+import React, { forwardRef, useState } from 'react';
+import PropTypes from 'prop-types';
+import { IconButton, InputAdornment, TextField } from '@mui/material';
+import { VisibilityOffOutlined, VisibilityOutlined } from '@mui/icons-material';
+
+const PasswordField = forwardRef(({ style, ...otherProps }, ref) => {
```

Comment:
Why do you extract here 'style' vs 'otherProps'

Reply:
you are right, is not needed. Removed!